src/xclim/core/units.py

```diff
@@ -64,28 +63,21 @@
 units = deepcopy(cf_xarray.units.units)
 # Changing the default string format for units/quantities.
 # CF is implemented by cf-xarray, g is the most versatile float format.
-# The following try/except logic can be removed when xclim drops support numpy <2.0.
-try:
-    units.formatter.default_format = "gcf"
-except UndefinedUnitError:
-    units.default_format = "gcf"
-# Switch this flag back to False. Not sure what that implies, but it breaks some tests.
-units.force_ndarray_like = False  # noqa: F841
-# Another alias not included by cf_xarray
-units.define("@alias percent = pct")
-
-# Default context.
-null = pint.Context("none")
-units.add_context(null)
+units.formatter.default_format = "gcf"
```

Comment:
`xclim` still supports `numpy>=1.23`; Won't this cause issues?

Reply:
The numpy condition was a confusion from me! The issue was actually with pint, which we already pin to the correct version.